src/components/AccountAssets/AccountAssets.tsx

```diff
@@ -12,12 +12,18 @@ type AccountAssetsProps = {
   accountId: AccountId
 }
 
+const TOKENS_SUPPORTING_CHAIN_NAMESPACES: ChainNamespace[] = [
```

Comment:
`CHAIN_NAMESPACE_SUPPORTS_TOKENS`?

Reply:
Absolutely, banged me head trying to name this one, much better 🧠 

1c753577cfc3ef1cf25cebcb8839cc331b779fcf